src/service/genesis_gen.h

```diff
@@ -420,6 +420,24 @@ namespace ccf
         {.status = CodeStatus::ALLOWED_TO_JOIN, .platform = platform});
     }
 
+    void trust_node_security_policy(
+      const std::optional<RawPolicy>& raw_security_policy,
+      const DigestedPolicy& host_data)
+    {
+      auto security_policies = tx.rw(tables.security_policies);
+      if (raw_security_policy.has_value())
+      {
+        LOG_INFO_FMT(
+          "Trusting node with policy {}", raw_security_policy.value());
+        security_policies->put(host_data, raw_security_policy.value());
```

Comment:
Same as for the constitution, we need to enforce that `host_data` is the digest of `raw_security_policy` here.

Reply:
We currently do a check on node start up, my understanding is that this isn't exposed publicly so would the check on node startup give us enough confidence not to need the check here? Or is it for defence in depth?

Reply:
I think this is more of a test for us, as this is hard to unit test, in case we ever break this.